### ISSUE TITLE
Typo on French "Hit regions and accessibility" page

### ISSUE THREAD

**Issue by cw118** [mdn/translated-content#5023]
MDN URL: https://developer.mozilla.org/fr/docs/Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility

#### What is the problem?

There's a small typo on the French [*Hit regions and accessibility* page](https://developer.mozilla.org/fr/docs/Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility) : "...pour les **navigteurs**..." should be "...pour les **navigateurs**..."

#### Specific section or headline?

[Moyen de repli](https://developer.mozilla.org/fr/docs/Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility#moyen_de_repli)

<details>
<summary>Page report details</summary>

* Folder: `fr/web/api/canvas_api/tutorial/hit_regions_and_accessibility`
* MDN URL: https://developer.mozilla.org/fr/docs/Web/API/Canvas_API/Tutorial/Hit_regions_and_accessibility
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/fr/web/api/canvas_api/tutorial/hit_regions_and_accessibility/index.md
* Last commit: https://github.com/mdn/translated-content/commit/500f444d23a7a758da229ebe6b9691cc5d4fe731
* Document last modified: 2022-03-16T16:52:18.000Z

</details>
